Sources/Commands/PinsStore.swift

```diff
@@ -10,6 +10,7 @@
 
 import Basic
 import SourceControl
+import PackageGraph
```

Comment:
It seems unfortunate to add this dependency just for this one thing. Can we make client convert this?